dockstore-integration-testing/src/test/java/io/dockstore/webservice/WebhookIT.java

```diff
@@ -127,28 +139,36 @@ public void testManualRefreshWorkflowWithGitHubApp() throws Exception {
         WorkflowsApi client = new WorkflowsApi(webClient);
 
         // Release 0.1 on GitHub - one new wdl workflow
-        List<Workflow> workflows = client.handleGitHubRelease(workflowRepo, "DockstoreTestUser2", "0.1", installationId);
+        List<Workflow> workflows = client.handleGitHubRelease(workflowRepo, "DockstoreTestUser2", "refs/tags/0.1", installationId);
         assertEquals("Should only have one service", 1, workflows.size());
 
         // Ensure that new workflow is created and is what is expected
         Workflow workflow = client.getWorkflowByPath("github.com/" + workflowRepo + "/foobar", "", false);
         assertEquals("Should be a WDL workflow", Workflow.DescriptorTypeEnum.WDL, workflow.getDescriptorType());
         assertEquals("Should be type DOCKSTORE_YML", Workflow.ModeEnum.DOCKSTORE_YML, workflow.getMode());
-        assertEquals("Should have one version 0.1", 1, workflow.getWorkflowVersions().size());
+        assertTrue("Should have a 0.1 version.", workflow.getWorkflowVersions().stream().anyMatch((WorkflowVersion version) -> Objects.equals(version.getName(), "0.1")));
 
         // Refresh
         workflow = client.refresh(workflow.getId());
         assertNotNull(workflow);
-        assertEquals("Should have two workflow versions: 0.1, 0.2 and 0.3", 3, workflow.getWorkflowVersions().size());
+        assertEquals("Should have four workflow versions: 0.1, 0.2 and 0.3, and master", 4, workflow.getWorkflowVersions().size());
```

Comment:
Isn't that a refresh on line 152? 
Couldn't someone else create new branches on DockstoreTestUser2/workflow-dockstore-yml and then break this?